cli/log.go

```diff
@@ -81,6 +81,10 @@ var LogSetLevel = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
+			return IncorrectNumArgs(cctx)
+		}
+
 		if !cctx.Args().Present() {
 			return fmt.Errorf("level is required")
 		}
```

Comment:
you've essentially duplicated the check here, one way or the other would be fine - either leave this one alone, or replace that existing `if` block with a new `IncorrectNumArgsWithHint` that preserves the hint (i.e. here it tells you what's missing, but in your version it just tells you that you've provided the wrong number of args)